examples/stable-diffusion/image_to_video_generation.py

```diff
@@ -164,7 +177,12 @@ def main():
         ),
     )
     parser.add_argument("--bf16", action="store_true", help="Whether to perform generation in bf16 precision.")
-
+    parser.add_argument(
+        "--num_frames",
+        type=int,
+        default=25,
+        help="The number of video frames to generate."
+    )
     args = parser.parse_args()
 
     from optimum.habana.diffusers import GaudiStableVideoDiffusionPipeline
```

Comment:
@wenbinc-Bin can you move the imoprt to the top?

Reply:
Done, Thanks.